src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift

```diff
@@ -58,7 +63,11 @@ class HealthCertificateService {
 			let healthCertificate = try HealthCertificate(base45: base45)
 
 			// check signature
-			if case .failure = signatureVerifying.verify(certificate: base45, with: [], and: Date()) {
+			if case .failure = signatureVerifying.verify(
+				certificate: base45,
+				with: dscListProvider.signingCertificates.value,
```

Comment:
The signingCertificates are fetched async, so at this point, it could happen that the signingCertificates are the default ones?

Reply:
Yes. We fetch them regularly when the app comes into foreground, that is already implemented in the service. We will then trigger a new validity check for all registered health certificates every time the list of signing certificates changes. (That is a task that is still open in JIRA.)

Reply:
Then we could change signingCertificates to not be a CurrentValueSubject?